includes/abstracts/abstract-wc-legacy-payment-token.php

```diff
@@ -0,0 +1,56 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Legacy Payment Tokens.
+ * Payment Tokens were introduced in 2.6.0 with create and update as methods.
+ * Major CRUD changes occurred in 2.7, so these were deprecated (save and delete still work).
+ * This legacy class is for backwards compatibility in case any code called ->read, ->update or ->create
+ * directly on the object.
+ *
+ * @version  2.7.0
+ * @package  WooCommerce/Classes
+ * @category Class
+ * @author   WooCommerce
+ */
+abstract class WC_Legacy_Payment_Token extends WC_Data {
+
+	/**
+	 * Read a tokem by ID (deprecated).
+	 */
+	public function read( $token_id ) {
+		_deprecated_function( 'WC_Payment_Token::read', '2.7', 'Init a token class with an ID.' );
```

Comment:
Let's soft deprecate them. I commented out the function for now since it is defined in your branch, but I have it on my todo list to review later.

Though we should probably open up a new ticket for after this sprint where we make sure everything is using the correct deprecation functions.